src/librustdoc/html/render.rs

```diff
@@ -1211,9 +1217,10 @@ impl DocFolder for Cache {
                 if !self.seen_mod {
                     if let Some(did) = did {
                         self.impls.entry(did).or_insert(vec![]).push(Impl {
-                            impl_: i,
-                            dox: attrs.value("doc").map(|s|s.to_owned()),
-                            stability: item.stability.clone(),
+                            // FIXME: would like to move the item here, but
```

Comment:
How come the bindings above were switched to `ref`? Wouldn't that mean you can avoid the `clone()` here?